fractal_tasks_core/tasks/apply_registration_to_image.py

```diff
@@ -285,10 +281,9 @@ def write_registered_zarr(
             of `build_pyramid`).
 
     """
-    # Read pixel sizes from zattrs file
-    pxl_sizes_zyx = extract_zyx_pixel_sizes(
-        f"{str(input_path / component)}/.zattrs", level=0
-    )
+    # Read pixel sizes from Zarr attributes
+    ngff_image_meta = load_NgffImageMeta(str(input_path / component))
```

Comment:
We already set this in line 128. Can we take the pixel size loading up there as well?